thrift/src/main/thrift/atoms/timeline.thrift

```diff
@@ -6,7 +6,7 @@ include "entity.thrift"
 include "../shared.thrift"
 
 struct TimelineItem {
-  1: required string title
+  1: optional string title
```

Comment:
I thought it was safe to change this, as long as you leave the numbering as it is?